### DISCUSSION TITLE
GoldHEN Cheats Manager v1.0.3 - Please update cheat database

### DISCUSSION THREAD

**Discussion by YumbaMumba** [GoldHEN/GoldHEN_Cheat_Repository#167] [Q&A]
Hi,

Would it be possible to update cheat database for GoldHEN Cheats Manager v1.0.3? I'm currently playing Callisto Protocol v1.14 and I can't use any cheats with it. It says it only works with version 1.12.

Thanks!

**Comment by bucanero** [comment]
See https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42
